src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  line-height: 14px;
+}
+
+.logo {
+  margin: auto 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: 100%;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    display: flex;
+    list-style: none;
+    padding: 0 10px;
+
+    &:first-child {
+      padding-left: 0;
+    }
+
+    &:last-child {
+      padding-right: 0;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+    font-size: 12px;
+    font-weight: 500;
+    color: $mainColor;
+    text-decoration: none;
+    &:hover {
+      color: $buttonColorBlue;
+    }
+  }
+}
+
+.is-active {
```

Comment:
for styling, instead of using class `is-active`, it would be nice to create a BEM modifier for `nav__link`